dll/ntdll/ldr/ldrutils.c

```diff
@@ -2292,6 +2292,11 @@ LdrpGetProcedureAddress(
             ImportName = RtlAllocateHeap(RtlGetProcessHeap(),
                                          0,
                                          Length);
+            if (!ImportName)
+            {
+                /* Return STATUS_INSUFFICIENT_RESOURCES since NT6.2 */
+                return STATUS_INVALID_PARAMETER;
```

Comment:
```suggestion
#if (NTDDI_VERSION >= NTDDI_WIN8)
                return STATUS_INSUFFICIENT_RESOURCES;
#else
                return STATUS_INVALID_PARAMETER;
#endif
```
or, using `_WIN32_WINNT`: `#if (_WIN32_WINNT >= _WIN32_WINNT_WIN8)`

@ThFabba @tkreuzer @learn-more Which one is better to use in ntdll?